app-mobilephone/heimdall/metadata.xml

```diff
@@ -1,8 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person" proxied="yes">
+		<email>marco@scardovi.com</email>
+		<name>Marco Scardovi</name>
+	</maintainer>
+	<maintainer type="project">
```

Comment:
proxied="proxy"

Reply:
Resolved

Reply:
I think `proxied="proxy"` goes on the proxy-maint maintainer element, and `proxied="yes"` goes on your own maintainer element.

Reply:
Oh, good point. I will take a look rn